src/FS.cpp

```diff
@@ -889,7 +889,15 @@ bool Directory::create (int mode /* = 0755 */)
 {
   // No error handling because we want failure to be silent, somewhat emulating
   // "mkdir -p".
-  return mkdir (_data.c_str (), mode) == 0 ? true : false;
+  Directory parent_dir = parent ();
+  if (! parent_dir.exists () )
```

Comment:
No space between the closing paren in the call to Directory::exists and the closing paren of the if.

Reply:
Good catch